src/apiProxy/apiConfigCreators.js

```diff
@@ -30,6 +30,34 @@ export function group(params) {
 }
 
 /**
+ * group communication endpoints
+ *
+ * accepts an `method` property in params
+ */
+export function mugcomm(params) {
```

Comment:
not sure if `mugcomm` is a term you're using internally, but most new code refers to mugs as `group`s and there's not really much advantage to abbreviating 'comm' here - I'd recommend `groupCommunications`, which maps semantically to the `/:urlname/communications/` endpoint paths